scoring/views.py

```diff
@@ -1380,7 +1380,7 @@ def get_context_data(self, **kwargs):
                             </ul>
                             """,
                         "clarifications": """
-                            <p>Marked mainly using  publicly available data on shared transport schemes.<a href="https://como.org.uk/" class="d-inline"> Coordinated Mobility's </a></p>
+                            <p>Marked mainly using  publicly available data on shared transport schemes from<a href="https://como.org.uk/" class="d-inline"> Coordinated Mobility</a></p>
```

Comment:
Bit odd that the space character is _inside_ the link element, rather than _before_ it.